src/components/texts/RepoCollections.vue

```diff
@@ -24,5 +36,18 @@ export default class RepoCollections extends Vue {
 	@Prop() private readonly collections!: TextCollection[];
 	@Prop() private readonly repoId!: string;
 	@Prop() private readonly queryParam!: string;
+	private page: number = 1;
+	private items: number[] = [5, 10, 15];
+	private perPage: number = 5;
 }
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+.row-select {
+	padding-right: 31px;
+}
+.rows {
```

Comment:
This needs to be named something else. You should be careful with class names. When you use grids in vuetify it just creates a div with the class row. Had you used rows in this you would have changed all the rows. This is usually the case with most css framworks as well e.g. bootstrap. I have added a screenshot so you can see what the rendered rows look like, it maybe helpful to look through the code that is rendered in the browers and read some documentation.

<img width="367" alt="Screen Shot 2020-03-02 at 1 06 41 PM" src="https://user-images.githubusercontent.com/16781171/75713459-102dd180-5c87-11ea-8900-c041d0738980.png">

Reply:
Also lets take the front size down just a little as well